dev/NavigationView/NavigationViewItemAutomationPeer.cpp

```diff
@@ -56,6 +56,20 @@ winrt::IInspectable NavigationViewItemAutomationPeer::GetPatternCore(winrt::Patt
     return result;
 }
 
+
+winrt::AutomationControlType NavigationViewItemAutomationPeer::GetAutomationControlTypeCore()
```

Comment:
Can you please add a test to validate. Thank you.

Reply:
Great idea! Added the test :)